test/compilable/extra-files/testcolor.sh

```diff
@@ -0,0 +1,43 @@
+compare()
+{
+    local actual expected
```

Comment:
Pedantic nit: `local` is not in POSIX (though it's supported pretty much everywhere)